hasher-matcher-actioner/hmalib/common/models/bank.py

```diff
@@ -127,21 +139,74 @@ class BankMember(DynamoDBItem):
     video, photo that a partner wants to match against.
     """
 
+    BANK_MEMBER_ID_PREFIX = "member#"
+
     bank_id: str
     bank_member_id: str
 
     content_type: t.Type[ContentType]
 
     # Will contain either the media_url (in case of photos / videos / pdfs) or
     # the raw_content in case of plain text.
-    media_url: str
-    raw_content: str
+    media_url: t.Optional[str]
```

Comment:
I was thinking content_type: Text with an actual text file. or a PDF. Where content_type might not have everything we need to determine how to process this.

Another case is urls, where the content_type is probably text, but instead of storing it in the URL, we'll want to write as raw_content.